tests/e2e/utils/RequestHandler.ts

```diff
@@ -0,0 +1,61 @@
+import { TestConstants, RequestType } from '..';
+import axios from 'axios';
+import querystring from 'querystring';
+import { injectable } from 'inversify';
+
+@injectable()
+export class RequestHandler {
+    async processRequest(reqType: RequestType, url: string, data?: string) {
```

Comment:
For what we should strict the axios opportunities?
If we need perform for example "PUT" method we should add this to the method.
But for me it looks like duplication.
In my opinion better to provide an object as for axios.

Reply:
So you propose to remove this class and use axios directly, right?

Reply:
not exactly, this class has an important and useful function - adding tokens.
I just propose don't define additional parameters and just provide ```{ ... }``` as for axios

Reply:
You suggested to separate this to single/multi user approach and handle that as injection. I started rewriting that and now I think we can absolutely get rid of this function, but it will mean using axios directly - and I think this can be OK.

Reply:
Ok, ignore this topic, I have just figured out that it is impossible.